windows.versions

```diff
@@ -0,0 +1,12 @@
+{
+  "windows": [
+    {
+      "version": "2.26.1",
+      "build": "1",
```

Comment:
I still find this pattern odd. Major.Minor.Patch for SymVer. I wonder if we should encode build as Major.Minor.Patch-Build if we do rebuilds. Then its just one string and makes a lot of sense. The Versions are fixed so the Major.Minor.Patch will never change, but the Build will increment month over month

Reply:
While we can definitely do that, it would lead to an additional step in various places wherein we would split this string into tokens and then decide various factors such as-
- If the rebuild for a particular version is required (It is strictly based on the build number)
- The version to be put in for the final image

There are other references as well for the same.
What do you suggest?

Reply:
I guess this is fine the way you have it. Just need to see the tooling side that produces the images. I'm worried we will forget to update "build" month over month but that isnt going to be any better/worse if we use a `-Build` suffix.

Reply:
Answered this in below comment- https://github.com/aws/aws-for-fluent-bit/pull/384#discussion_r923690961